### DISCUSSION TITLE
continuous days

### DISCUSSION THREAD

**Discussion by shackelford** [juftin/camply#20] [Ideas]
this tool is amazing!  is there a way to search by the number of nights within a window, e.g. 5 nights between june 15 and august 15?  is it possible to sort the results by site and then by date to know which site has the most continuous days available?  thanks!

**Comment by juftin** [comment]
@shackelford this is a great idea, and definitely something I've had my eye on since day 1. Unfortunately, for now, the answer is ~no~ not yet. 

To implement this I would just need to find a way to perform some additional grouping and filtering on the campsites returned. I'm already performing some sorting and grouping via _pandas_ so this shouldn't be terribly difficult. 

I'll see what I can do in the coming weeks to come up with something. I imagine it'll probably look something like this on the CLI:
```
camply campsites \
    --start-date 06-15-2021 \
    --end-date 08-15-2021 \
    --nights 5 \
    --rec-area 1234
```

I'll keep you updated here on my progress. Thank you very much for your idea, I really appreciate it. I'm so glad you like the tool!

**Reply by shackelford** [reply]
awesome, thanks again
